route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -161,17 +177,22 @@ namespace route_following_plugin
 
         while (planned_time < min_plan_duration_ && i < latest_maneuver_plan_.size())
         {
+            ROS_DEBUG_STREAM("Checking maneuver id " << i);
+            ROS_ERROR_STREAM("Checking maneuver id " << i);
             //Ignore plans for distance already covered
             if (GET_MANEUVER_PROPERTY(latest_maneuver_plan_[i], end_dist) < current_downtrack)
             {
+                ROS_DEBUG_STREAM("Skipping maneuver id " << i);
+                ROS_ERROR_STREAM("Skipping maneuver id " << i);
+
                 ++i;
                 continue;
             }
             if(planned_time == 0.0){
                 //update start distance of first maneuver
                 setManeuverStartDist(latest_maneuver_plan_[i], current_downtrack);
             }
-            planned_time += getManeuverDuration(latest_maneuver_plan_[i], epsilon_).toSec();
+            planned_time += getManeuverDuration(latest_maneuver_plan_[i], 0.0001).toSec();
```

Comment:
Seems like the epsilon_ constant got replaced with its actual value? Probably best to change it back since the epsilon_ variable at least had a comment explaining its use.